libpod/state_test.go

```diff
@@ -281,6 +281,95 @@ func TestAddCtrDepInPodFails(t *testing.T) {
 	})
 }
 
+func TestAddCtrDepInSameNamespaceSucceeds(t *testing.T) {
```

Comment:
So many namespace tests, I wonder if it would make sense to create a new file for them and move them out of this one.

Reply:
The file probably ought to be cut up into four or five smaller ones, not just two, given it's already hit 3500 lines. Once things stabilize a bit more, I'll look into doing some refactoring like that.